test/common/index.js

```diff
@@ -45,6 +45,32 @@ const isMainThread = (() => {
   }
 })();
 
+// Check for flags.
+if (module.parent && isMainThread && process.argv.length === 2) {
+  // The copyright note is relatively big and the flags could come afterwards.
+  const bytesToRead = 1500;
+  const buffer = Buffer.allocUnsafe(bytesToRead);
+  const fd = fs.openSync(module.parent.filename, 'r', 0o666);
```

Comment:
```suggestion
  const fd = fs.openSync(module.parent.filename, 'r');
```

The `mode` argument would be ignored if the file is not being created by this call.